ccf_transaction_processor/Readme.md

```diff
@@ -0,0 +1,46 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+## Microsoft CCF based PDO Transaction Processor
+
+This folder contains software for PDO transaction processor (TP) based on Microsoft's CCF blockchain.
+The software is located under transaction_processor/. The folder CCF/ points to CCF tag 0.7.1 which is
+included as a submodule under the PDO repo. The TP software is written and tested for CCF tag 0.7.1.
+Compatability with other CCF versions is not guaranteed.
+
+The TP must viewed as a CCF application. Documentation for building and deploying CCF applications
+can be found at https://microsoft.github.io/CCF/. The CCF legder that stores the PDO TP registries is
+encrypted, and is accessible only within CCF encalves. Currently PDO/CCF combination is supported
+only under the virtual enclave mode for both PDO and CCF. (set env variable SGX_MODE=SIM for PDO &
```

Comment:
I think it would be really helpful for the reader to break up this README into a few sections. I would be reading this document to learn about CCF but also how to set up CCF, so I would place the actual instructions for configuring CCF in their own separate "Setup" section.

Reply:
I agree we need to vastly expand this document, this is tracked in Issue 264. For now, this is a very quick document to get going on PDO/CCF for some one who knows both PDO & CCF.

Reply:
thank you for the note. The current document is about just one page long. I don't see the need to modularize this one-page document given the document is a "whole package". Definitely, when we expand this.

Reply:
The parts that are actual setup wouldn't change once we expand this document, would they? Not sure how modularization is affected by the length of the document as it is now. It seems like you are planning to make more changes to this document in the near future, so I look forward to that PR!